cvat-core/src/enums.js

```diff
@@ -33,6 +33,20 @@
         COMPLETED: 'completed',
     });
 
+    /**
+     * Task dimension
+     * @enum
+     * @name TaskDimension
+     * @memberof module:API.cvat.enums
+     * @property {string} DIMENSION_2D '2d'
+     * @property {string} DIMENSION_3D '3d'
+     * @readonly
+     */
+    const TaskDimension = Object.freeze({
```

Comment:
Let's use unified name here **DimensionType** (like in python code)

Reply:
Made changes accordingly